packages/sui-theme/src/settings-compat-v7/_components.scss

```diff
@@ -182,21 +182,23 @@ $size-dropdown-icon: 18px !default;
 $z-dropdown-menu: 1 !default;
 
 // Dropdown basic
-$fw-dropdown-basic-button: inherit !default;
+$bdr-dropdown-basic-menu: 0 !default;
 $c-dropdown-basic-button-hover: $c-accent !default;
 $c-dropdown-basic-button-icon: $c-accent !default;
 $c-dropdown-basic-list-link: $c-accent !default;
+$fw-dropdown-basic-button: inherit !default;
 
 // Dropdown user
 $bd-dropdown-user-last-item: 1px solid $c-gray-light !default;
+$bdr-dropdown-user-menu: 0 !default;
```

Comment:
```suggestion
$bdrs-dropdown-user-menu: 0 !default;
```

Reply:
`$bdrs` for broder radius (`$bdr` is border right)

Reply:
Ok! Thanks!